packages/syft/src/syft/service/action/action_service.py

```diff
@@ -291,6 +300,10 @@ def _user_code_execute(
 
         if not override_execution_permission:
             input_policy = code_item.input_policy
+            if input_policy is None:
```

Comment:
I added an error handling here to make the `test_code_accept_deny` passed. I think it looks right, right? @koenvanderveen